### PR TITLE
[ADD] base.group_portal for external users

### PR DESCRIPTION
**Pull request by ntsirintanis**

### CODE REVIEW

**Comment by ntsirintanis** [comment]
@hbrunn

File: trp_external_user/security/res_groups.xml [hunk 1]
```diff
@@ -3,6 +3,9 @@
     <record id="group_external_user" model="res.groups">
         <field name="name">External users</field>
         <field name="category_id" ref="module_category_external_users"/>
+        <field
+                name="implied_ids"
```
Comment (NL66278) [inline trp_external_user/security/res_groups.xml]: Correct indentation

**Review by NL66278 — CHANGES_REQUESTED** [review]
Mostly OK, but needs improved indentation

**Comment by ntsirintanis** [comment]
@NL66278 it is now green.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**